src/App.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import "./styles.css";
+import PokeCard from "./components/PokeCard";
 
 export default function App() {
   return (
     <div className="App">
-      <h1>Welcome to your second mini challenge!</h1>
+      <PokeCard />
```

Comment:
It would be better to inject mock data here instead of inside the component.
Advantages:

- Improve your Unit testing
- Avoid DRY when using the component in different parts of your app